generator.sh

```diff
@@ -48,12 +48,27 @@ fi
 hostname=""
 bundle_identifier=""
 
+bundle_regex="^[a-zA-Z]+(\.?[a-zA-Z]+[0-9a-zA-Z\_]*)+$"
+
 while [ -z "$hostname" ]; do
     read -p'--> On Android, which host would you like to use for deep linking? (eg. www.mobify.com) ' hostname
 done
 
 while [ -z "$bundle_identifier" ]; do
     read -p"--> Which iOS Bundle Identifier and Android Package Name would you like to use? Begin with 'com.mobify.' to use HockeyApp. (eg. com.mobify.app) " bundle_identifier
+
+    if [[ "$bundle_identifier" =~ $bundle_regex ]]; then
+        read -p"--> Continue with the iOS Bundle Identifier and Android Package Name $bundle_identifier? (y/n) " -n 1 -r
+        echo
+        if [[ ! $REPLY =~ ^[Yy]$ ]] ; then
+            bundle_identifier=''
+        fi
+    else
+        bundle_identifier=''
+        echo '--> Invalid package name'
```

Comment:
We should also change the 'to use HockeyApp' bit to: 'to use the Mobify provisioning' ? Thoughts on this?